.vimrc

```diff
@@ -146,6 +125,15 @@ autocmd CursorHold * silent call CocActionAsync('highlight')
 " Symbol renaming.
 nmap <leader>rn <Plug>(coc-rename)
 
+" Add (Neo)Vim's native statusline support.
+" NOTE: Please see `:h coc-status` for integrations with external plugins that
+" provide custom statusline: lightline.vim, vim-airline.
+set statusline^=%{coc#status()}%{get(b:,'coc_current_function','')}
```

Comment:
I re-copied this from the example on the coc.nvim GH - this gives you stuff like the percent of files indexed by rust-analyzer and such